tools/mo/openvino/tools/mo/front/tf/MapFNTransformation.py

```diff
@@ -5,15 +5,15 @@
 
```

Comment:
please update `Supported_Frameworks_Layers.md` with all new mentioned operations and make a node "Supported only when it is part of a sub-graph of the special form"

Reply:
Done.